### PR TITLE
[dom daint eiger pilatus tsa] Add recipe for ReFrame CSCS tests version 22.07

### PR DESCRIPTION
**Pull request by jenkins-cscs**

### CODE REVIEW

**Comment by teojgo** [comment]
Ok to test

**Comment by lucamar** [comment]
retest this please

**Review by ekouts — APPROVED** [review]

File: easybuild/easyconfigs/r/reframe-cscs-tests/reframe-cscs-tests-22.07.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+name = 'reframe-cscs-tests'
+version = '22.07'
+
+homepage = 'https://github.com/eth-cscs/cscs-reframe-tests'
+description = 'ReFrame test suite of CSCS.'
+
+toolchain = SYSTEM
+
+dependencies = [('reframe', '3.11.1')]
```
Comment (lucamar) [inline easybuild/easyconfigs/r/reframe-cscs-tests/reframe-cscs-tests-22.07.eb:13]:
```suggestion
dependencies = [('reframe', '3.12.0')]
```

Comment (lucamar) [reply easybuild/easyconfigs/r/reframe-cscs-tests/reframe-cscs-tests-22.07.eb:13]: Is it worth updating the dependency to the latest `reframe` module available in production?

**Pull request merged**